e2e-test/README.md

```diff
@@ -0,0 +1,116 @@
+# End-to-End Tests
+
+The end-to-end tests use [Puppeteer](https://github.com/puppeteer/puppeteer/) and [Jest](https://jestjs.io/en/) to connect to a running instance of the application and use it through a browser like a normal user would.
+
+## Installation
+```bash
+cd e2e-test
+yarn install
+```
+
+Puppeteer includes a bundled Chromium executable, but not all the various Chrome dependencies.
+This is not generally an issue in development environments where Chrome is already installed.
+Be aware of this when setting up CI or Docker images though.
+
+## Running Tests
+You will need a running instance of the app, both the `web` and `flask` components.
+Assuming the web app is running at `http://localhost:8080`:
```

Comment:
I suspect all references to `localhost:8080` will need to be changed to `localhost:8085`